engine/src/main/java/org/terasology/world/chunks/localChunkProvider/LocalChunkProvider.java

```diff
@@ -92,6 +92,7 @@
     private static final Logger logger = LoggerFactory.getLogger(LocalChunkProvider.class);
     private static final int UNLOAD_PER_FRAME = 64;
     private final EntityManager entityManager;
+    private final BlockingQueue<Chunk> readyChunks = Queues.newLinkedBlockingQueue();
```

Comment:
I think `preparedChunks` sounds better.(open question)
ReadyChunks is confusing name.

Reply:
`readiedChunks` maybe? To me readied and prepared would mean about the same, unless we're using either "ready" or "prepare" as stage names or something during chunk gen :-)